sys/include/net/gnrc/tcp.h

```diff
@@ -34,6 +34,13 @@
 extern "C" {
 #endif
 
+#ifndef GNRC_TCP_NO_TIMEOUT
```

Comment:
I think it shouldn't be possible to overwrite this, otherwise we have a miss-match with `SOCK_NO_TIMEOUT`

**edit** ah now I see why you did it like this - better add a note that this is only made overwritable for the unit test, so people won't get funky ideas :wink:

Reply:
Thats why I did not make it configurable via kconfig. But I'll add a comment